### PR TITLE
fix rl-report issues

### PR DESCRIPTION
**Pull request by KivenChen**
According to rl-report mentioned  in #137, three changes were added:
- rename `ci/azure_pipeline.yml` to `ci/azure_pipelines.yml`
- Added CODE_OF_CONDUCT from Hyperledger project
- Added NOTICE from Hyperledger project

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Hyperledger Fabric
```
Comment (dexhunter) [inline NOTICE:1]: I think we should use `Hyperledger Fabric Python SDK` instead of `Hyperledger Fabric`

File: NOTICE [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+Copyright [2016-2020] contributors to Hyperledger Fabric
```
Comment (dexhunter) [inline NOTICE:2]: Copyright [2016-2021] contributors to Hyperledger Fabric Python SDK

**Review by dexhunter — DISMISSED** [review]
thank you for the PR, please check the comments

**Comment by KivenChen** [comment]
> thank you for the PR, please check the comments

Problem resolved 🚀

**Review by dexhunter — DISMISSED** [review]
lgtm

**Comment by dexhunter** [comment]
@KivenChen hi, could you keep the original `azure-pipeline.yml`, we will delete the old ones after this PR merged. Thanks

**Pull request merged**